uniffi_bindgen/src/templates/RustBuffer.rs

```diff
@@ -0,0 +1,8 @@
+// Everybody gets basic bytebuffer allocation and freeing, since it's needed
+// for passing complex types over the FFI.
+#[no_mangle]
+pub extern "C" fn {{ ci.ffi_bytebuffer_alloc().name() }}(size: u32) -> uniffi::ffi_support::ByteBuffer {
```

Comment:
A minor suggestion seeing the generated code, I wonder if it would be worth namespacing this re-exported modules under something like `uniffi::deps::ffi_support` or similar, to make it a little more obvious what's going on when you see this in the generated code.